meshtastic/mesh.proto

```diff
@@ -558,6 +558,27 @@ enum HardwareModel {
    */
   HELTEC_CAPSULE_SENSOR_V3 = 65;
    
+   /*
+   * Heltec Vision Master T190 with ESP32-S3 CPU, and a 1.90 inch TFT display
+   */
+  HELTEC_VISION_MASTER_T190 = 66;
+   
+   /*
+   * Heltec Vision Master E213 with ESP32-S3 CPU, and a 2.13 inch E-Ink display
+   */
+  HELTEC_VISION_MASTER_E213 = 67;
+   
+   /*
+   * Heltec Vision Master E290 with ESP32-S3 CPU, and a 2.9 inch E-Ink display
+   */
+  HELTEC_VISION_MASTER_E290 = 68;
+   
+   /*
+   * Heltec Mesh Node T114 borad with nRF52840 CPU, and a 1.14 inch TFT display, Ultimate low-power design,
```

Comment:
Spelling issue with board

Reply:
Excuse me, I don't seem to have found any problems, how should I make changes? Did you mean a product can be named in `Mesh`?

Reply:
Sorry I should have clarified. Comment says "borad" instead of board

Reply:
Ohhh...  very sorry, I'm an idiot. Already fixed.

Thank you in advance for review again.